scilpy/tractograms/tractogram_operations.py

```diff
@@ -47,6 +68,47 @@ def shuffle_streamlines(sft, rng_seed=None):
     return shuffled_sft
 
 
+def shuffle_streamlines_orientation(sft, rng_seed=None):
+    """
+    Shuffle the orientation of the streamlines. The streamlines are not
+    reversed, only the orientation of the points within the streamlines is
+    shuffled.
+
+    Parameters
+    ----------
+    sft: StatefulTractogram
+        The tractogram that will have its streamlines' orientation shuffled.
+    rng_seed: int
+        Random seed.
+
+    Returns
+    -------
+    shuffled_sft: StatefulTractogram
+        The shuffled tractogram.
+    """
+    if sft.data_per_point is not None and len(sft.data_per_point) > 0:
+        logging.warning('Shuffling streamlines orientation. DDP will be '
+                        'lost.')
+
+    rng = np.random.RandomState(rng_seed)
+    streamlines = sft.streamlines
```

Comment:
No need for that variable. sft.streamlines is only four letters more.

Reply:
@frheault ?